kf/bpf.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"syscall"
```

Comment:
can line 26 be removed now?

Reply:
We are using syscall here - line# 231 b.Cmd.Process.Signal(syscall.SIGTERM)